clang/test/Driver/sanitizer-ld.c

```diff
@@ -8,7 +8,7 @@
 // RUN:     --sysroot=%S/Inputs/basic_linux_tree \
 // RUN:   | %{filecheck} --check-prefix=CHECK-ASAN-LINUX
 //
-// CHECK-ASAN-LINUX: "{{(.*[^-.0-9A-Z_a-z])?}}ld{{(.exe)?}}"
+// CHECK-ASAN-LINUX: "{{(.*[^-.0-9A-Z_a-z])?}}ld"
```

Comment:
Is this not needed if you are running this test on Windows?

Reply:
I believe `--target=i386-unknown-linux` still make it no-exe even on WIndows.

Reply:
Looks like all windows bots are OK https://lab.llvm.org/buildbot/#/builders/193/builds/4341

Reply:
It passes on some, and fail on others https://buildkite.com/llvm-project/github-pull-requests/builds/132244